neXtProt.pm

```diff
@@ -45,6 +45,10 @@ limitations under the License.
  This plugin is only suitable for small sets of variants as an additional 
  individual remote API query is run for each variant.
 
+ The neXtprot_headers.txt file is a requirement for running this plugin and is found alongside the plugin in the
```

Comment:
Minor change: filename is neXtProt_headers.txt

Reply:
Good spot! Fixed the name.

Reply:
Thanks!